src/pages/LoginScreen/LoginScreen.jsx

```diff
@@ -0,0 +1,143 @@
+//Remove this file
+import { useState } from "react";
+import "../../../src/assests/CSS/loginPage.css"
+import useAuth from "../../context/AuthContext";
+import { SampleNavBar } from "../../layouts/SampleNavBar";
+import axios from "axios";
+import UserService from "../../services/UserService";
+import palliLogo from "../../../src/assests/images/dckap-palli-tamil-logo-1 1.png"
+import loginLogo from "../../../src/assests/images/Seminar-bro 1.png";
+import { EyeInvisibleOutlined, EyeTwoTone } from "@ant-design/icons";
+import { isEmailValid, isPasswordValid } from "../../utils/loginPageValidation";
+import { Input } from "antd";
+import PasswordRequirements from "../../components/PasswordRequirement";
+import { SampleButton } from "../../components/SampleButton";
+const LoginScreen = () => {
+  //while typing the password show password check list to user
+  const [showPasswordCheckList,setshowPasswordCheckList] = useState(false);
+
+  //this state collect the user details 
+  const [loginUserData, setloginUserData] = useState({
+    email: "",
+    password: "",
+  });
+
+  //user give wrong value then show error message 
+  const [error, setError] = useState({});
+
+  
+  //handlechange used to set user email and password in logindata state
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setloginUserData({ ...loginUserData, [name]: value });
+    if (error[name]) delete error[name];
+  };
+
+  //this validate function used to user give the email and password vaild or not
+  const validate = (loginUserData) => {
+    let errors = {};
+    let isVaild = true;
+
+    if (isEmailValid(loginUserData.email)) {
+      errors.email = isEmailValid(loginUserData.email);
+      isVaild = false;
+    }
+    if (isPasswordValid(loginUserData.password)) {
+      errors.password = isPasswordValid(loginUserData.password);
+      isVaild = false;
+    }
+
+    setError(errors);
```

Comment:
setState should be in the same file here, but all the other logics should be moved to utils